### ISSUE TITLE
Improving the script to train and run neural networks

### ISSUE THREAD

**Issue by andrefdre** [DanielCoelho112/AutoMec-AD#199]
Currently, during training, the augmentation performs augmentation and preprocessing using two different libraries. It would be interesting to change to use torchvision transforms for everything.
Another feature to improve is to save the image size using during training in the model yaml config so when running the model it gets image size information automatically. 
Lastly, it would be interesting to normalize the images in the preprocessing and for that get the mean and standard deviation in the dataset.

**Comment by andrefdre** [comment]
The script to calculate the mean and std from the dataset was created. Now before training networks, it's necessary to run this script in order to use the calculated parameter in the normalization.
```
rosrun prometheus_driving create_statistics.py -d *dataset name*
```

The image size and the normalization parameter are now stored when saving the model for later use during evaluation. 

The augmentation was changed to use torchvision for easier control and better code readability.

**Comment by andrefdre** [comment]
The code was commented to have easier readability, and the same alterations were performed for semantic segmentation.
